src/webgpu/api/operation/memory_sync/buffer/ww.spec.ts

```diff
@@ -83,12 +83,54 @@ g.test('two_draws_in_the_same_render_pass')
     a storage buffer. The second write will write 2 into the same buffer in the same pass. Expected
     data in buffer is either 1 or 2. It may use bundle in each draw.`
   )
-  .unimplemented();
+  .params(pbool('useBundle'))
```

Comment:
Now that I'm looking at the implementation, I think we should test at least these three options:
- same pass, no bundle
- same pass, different bundles
- same bundle

Reply:
Agreed. I was meant to test same pass, no bundle, same pass, one bundle, same pass, two separate bundles. But I wrote it wrongly. And I ignored same bundle (definitely one pass). 
Now I added all of them. Note that I separated the tests two-draws-in-the-same-bundle from the original one. I tried to merge into the same one, but it requires unless and multiple ifs, making the code logic in a mess. So I think it is better to write a standalone test for same-bundle test.